src/Carbon/Carbon.php

```diff
@@ -1984,6 +1984,58 @@ public function addYear($value = 1)
     }
 
     /**
+     * Add years to the instance with no overflow of months
```

Comment:
Like this? 
```
public function addYearsNoOverflow($value)
{
    return $this->addMonths($value * static::MONTHS_PER_YEAR);
}
```
is it not already implied that the user requires the overflow if they call this method? It wouldn't make sense for this method to be called but the months to not overflow?